examples/plotting.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"math"
+	"os"
+
+	"gonum.org/v1/plot"
+	"gonum.org/v1/plot/plotter"
+	"gonum.org/v1/plot/vg"
+	"gonum.org/v1/plot/vg/draw"
+
+	"github.com/tobgu/qframe"
+	qfnum "github.com/tobgu/qframe/contrib/gonum"
+)
+
+func Sin(n int, max float64) qframe.QFrame {
+	xs := make([]float64, n)
+	ys := make([]float64, n)
+	for i := 0; i < n; i++ {
+		xs[i] = max / float64(n) * float64(i)
+		ys[i] = math.Sin(xs[i]) * 100
+	}
+	return qframe.New(map[string]interface{}{
+		"X": xs,
+		"Y": ys,
+	})
+}
+
+func main() {
```

Comment:
perhaps turn this into a real `Example` instead of a binary?
this way we run it as part of `qframe`'s tests.